sdk/spring/spring-cloud-azure-feature-management/src/main/java/com/azure/spring/cloud/feature/management/FeatureManagementConfiguration.java

```diff
@@ -27,7 +34,19 @@ class FeatureManagementConfiguration {
      */
     @Bean
     FeatureManager featureManager(ApplicationContext context,
```

Comment:
Why there are two contexts, one `context` and one `appContext`, what's their differences?